pkg/apis/postgres-operator.crunchydata.com/v1beta1/crunchy_bridgecluster_types.go

```diff
@@ -79,6 +88,21 @@ type CrunchyBridgeClusterSpec struct {
 	Storage resource.Quantity `json:"storage"`
 }
 
+type CrunchyBridgeClusterRoleSpec struct {
+	// The name of this PostgreSQL role. The value may contain only lowercase
+	// letters, numbers, and hyphen so that it fits into Kubernetes metadata.
+	// The above is problematic for us as Bridge has a role with an underscore.
+	// TODO: figure out underscore dilemma
+	// +kubebuilder:validation:Pattern=`^[A-Za-z][A-Za-z0-9\-_ ]*[A-Za-z0-9]$`
+	// +kubebuilder:validation:Type=string
+	Name string `json:"name"`
```

Comment:
❓ Is this effectively the `name` field of [ClusterRole](https://docs.crunchybridge.com/api/cluster-role#cluster-role)? Can we drop regex validation and let the API tell us what's valid?

```suggestion
	// Name of the role within Crunchy Bridge.
	// More info: https://docs.crunchybridge.com/concepts/users
	Name string `json:"name"`
```